api/src/endpoints/neo4j.js

```diff
@@ -28,10 +28,10 @@ const neo4jRoutes = express.Router();
 
 const fetchWith = async (req, res, queryHandler) => {
   const { id } = req.params;
-  const { model, version, limit, full, searchTerm, componentTypes } = req.query;
+  const { model, modelShortName, version, limit, full, searchTerm, componentTypes } = req.query;
```

Comment:
Currently we don't have a good system for syncing a model's database label (HumanGem) and a model's shortName (Human-GEM). For example if you look at [this line](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/feat/explain-large-number-of-missing-cytosol-reactions/api/src/neo4j/queries/gene.js#L57), we are manually mapping the short name into the database label.

I think that for now I would prefer using a a similar manual conversion (construct `modelShortName` from `model`) to avoid sending extra parameters between the frontend and api. This conversion should probably be done in the `api/src/neo4j/queries/map.js` file instead of here so it's not sent to the other functions that currently do not need it.

For the long term I think we should discuss and establish a better system for syncing these varieties.

Reply:
@inghylt I was thinking the `modelShortName` should be removed from this file too. Do we need it for something else?

Reply:
Ugh sorry that I missed that. Fixed with commit 55c4c03c154645ea82ef17cb14499b1de478e8c2

Reply:
No worries, thank you for fixing it!